nussl/ml/networks/separation_model.py

```diff
@@ -8,7 +8,25 @@
 
 from . import modules
 from ... import __version__
+import copy
 
+def _remove_cache_from_tfms(transforms):
+    from ... import datasets
+    transforms = copy.deepcopy(transforms)
+
+    if isinstance(transforms, datasets.transforms.Compose):
+        for t in transforms.transforms:
+            if isinstance(t, datasets.transforms.Cache):
+                transforms.transforms.remove(t)
+
+    return transforms
+
+
+def _prep_metadata(metadata):
+    metadata = copy.deepcopy(metadata)
```

Comment:
Maybe a 1 line docstring for each of these funcs, now that their context has changed.